### PR TITLE
Update AFDX sample - Front Door Standard/Premium (Preview) with API Management

### PR DESCRIPTION
**Pull request by johndowns**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Switch to using HTTPS for origin forwarding protocol.
* Update README.md to clarify behaviour.

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Review by wachabaha — APPROVED** [review]

**Comment by wachabaha** [comment]
- @johndowns image

File: 201-front-door-standard-premium-api-management-external/README.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ To access the API proxy URL, get the hostname from the output named `frontDoorEn
 
 If you see an error page, wait a few minutes and try again.
```
Comment (wachabaha) [inline 201-front-door-standard-premium-api-management-external/README.md:56]:
```suggestion
 If you see an error page, wait a few minutes and try again.
**_[](https://github.com/Azure/azure-quickstart-templates/pull/8884#issuecomment-799236097)_**
```

**Review by wachabaha — APPROVED** [review]

**Review by wachabaha — COMMENTED** [review]
https://github.com/Azure/azure-quickstart-templates/pull/8884/files

**Pull request merged**